hedera-node/configuration/previewnet/bootstrap.properties

```diff
@@ -1,6 +1,3 @@
 ledger.id=0x02
 contracts.chainId=297
 bootstrap.genesisPublicKey=c249a323c878f5b5e2daccda6d731e6fdc32f870228d1cd4fae559d947dbc36c
-hedera.recordStream.recordFileVersion=6
-hedera.recordStream.signatureFileVersion=6
-contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
```

Comment:
Please restore this.
We want to keep actions on in previewnet and local node for the mean time.

I think that affects the `preprod` and `dev/bootstrap.properties` also right?

Reply:
Per my table above (PR comment) preprod does **not** have actions turned on now.  Only previewnet and dev (whatever that is).  Also, AFAIK actions are also **not** on in local node which I believe has it's own (templating) mechanism for creating `bootstrap.properties` from a json file, and the key files over there, `configs/*.json` do not specify contract actions sidecar be enabled.